Makefile

```diff
@@ -0,0 +1,12 @@
+DIST := dist
+FILEBASE := universal-search
+VERSION := `cat install.rdf | tr '\n' ' ' | sed "s/.*\<em:version\>\([^\<]*\).*/\1/"`
```

Comment:
How about `sed "s/.*<em:version>\(.*\)<\/em:version>.*/\1/"` instead? Simpler, does the same stuff.